vllm/entrypoints/openai/api_server.py

```diff
@@ -687,6 +688,11 @@ async def do_rerank_v2(request: RerankRequest, raw_request: Request):
 
 if envs.VLLM_SERVER_DEV_MODE:
 
+    @router.get("/server_info")
+    async def show_server_info(raw_request: Request):
+        server_info = {"vllm_config": str(raw_request.app.state.vllm_config)}
```

Comment:
```suggestion
        server_info = {"vllm_config": base(raw_request).vllm_config)}
```

Also should we serialize the config into JSON?

Reply:
VllmConfig is  currently not JSON serializable， so I use "str(raw_request.app.state.vllm_config)". I've looked at the __str__() of VllmConfig, the returned information is well-organized and sufficient. I think users can obtain this string and parse it themselves easily.

Reply:
Ok, let's just apply the suggestion without the serialization then

Reply:
Ok, thank you for your suggestions. I set `state.vllm_config = vllm_config` in `init_app_state`, and for now I haven’t added vllm_config to `OpenAIServing`, so I  prefer to use `raw_request.app.state.vllm_config` rather than the `base` function here.

Reply:
Oh, I see. Then it should be fine as is